tests/resources/builder/build_split/expected_no_td.ipynb

```diff
@@ -906,6 +886,14 @@
     "## Outcomes"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "1757fda1",
+   "metadata": {},
+   "source": [
+    "See our documentation on [analyzing outcomes](https://epic-open-source.github.io/seismometer/user_guide/index.html#outcomes) for more details."
```

Comment:
Should we add a sentence before "See ..." just to briefly introduce Outcomes (same for Fairness) as these concepts are relatively new.

Reply:
finc